collections/_infographics/emissions/emissions-eu-trends.md

```diff
@@ -13,4 +13,38 @@ data-orig:
+Emissions in the energy sector come primarily from electricity and heat production. These emissions can be reduced through energy savings and the development of renewable and low-carbon energy sources. They also come from leakages from fuel extraction, fuel production and treatment and oil processing.
+* __Transport:__ Transport emissions increased by 33% compared to 1990 to 968 million tonnes of CO<sub>2</sub>eq per year. The detailed graph on the right shows a temporary drop in emissions after 2007 due to the global financial crisis and the subsequent economic recession. Since 2013, emissions can be seen to have risen again. GHG emissions from transport are primarily from the combustion of fossil fuels in the engines of road transport vehicles. (In 2019, 81% of all emissions from the transport sector were from transport, with aviation accounting for 15%.)
+It is possible to reduce transport emissions by switching to alternative fuels (e.g., biomethane, CNG, hydrogen or electricity with a simultaneous energy transformation), increasing the share of public transport (train and bus) and reducing the number of vehicles on the road. The volume of road transport can be reduced by increasing vehicle occupancy (carpooling) or generally decreasing the need for transportation (e.g. teleworking).
+* __Industrial processes:__ Emissions from industrial processes have fallen by 26% since 1990 to 343 million tonnes of CO<sub>2</sub>eq per year. These are emissions from processes that release greenhouse gases into the atmosphere other than through the combustion of fossil fuels - in the production of chemicals, cement, iron and steel, etc.  
+For example, emissions from cement production have fallen by almost a quarter since the 1990s. There has also been a decline in the chemical sector, with emissions from ammonia production falling by 34% and nitric acid production falling by 89% (they were still 46 Mt CO<sub>2</sub>eq in 1990). In contrast, there is a slight increase in emissions from petrochemicals. There are also decreases in iron and steel production (down 35%) and aluminium production (down 85%). Since the 1990s, emissions from the use of refrigerants have increased significantly, replacing the ozone-depleting substances previously used, which are now regulated under the Montreal Protocol. These emissions rose from zero to 88.4 Mt CO<sub>2</sub>eq in 2014. Since then, these same emissions have been declining, reaching 80.3 Mt CO<sub>2</sub>eq in 2019.
+* __Residential, institutional and agricultural combustion:__ Emissions fell by 27% compared to 1990 to 519 million tonnes of CO<sub>2</sub>eq per year. These emissions are from heating and hot water (unless a heating plant supplies energy), cooking, etc. It also includes fuel combustion in agriculture and forestry.
```

Comment:
comma. change
Emissions fell by 27% compared to 1990 to 519 million tonnes of CO<sub>2</sub>eq per year.
to
Emissions fell by 27% compared to 1990, to 519 million tonnes of CO<sub>2</sub>eq per year.

Change
These emissions are from heating and hot water (unless a heating plant supplies energy), cooking, etc. It also includes fuel combustion in agriculture and forestry.
to
These emissions arise from heating and hot water (unless a heating plant supplies energy), cooking, etc. They also include fuel combustion in agriculture and forestry.

Reply:
Thanks. I updated both:
1. added the comma after "1990"; and
2. updated the two sentences.